### DISCUSSION TITLE
Unit 5 religious restrictions?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/2nd-semester-introduction-to-computer-science#374] [Q&A]
I wanted to ask if anyone has had a student who could not participate in Unit 5 because of religion? There should be about 0.1% of our students who cannot participate (based on 2017 Pew data.)

**Comment by Dan-TEALS** [comment]
Sorry to just now be asking this but are you thinking religious restrictions regarding secular music? Note that unit 5 is already an optional unit in the curriculum, so it could easily be skipped but this is good consideration from a DEI standpoint.

**Reply by jdonwells** [reply]
Some Muslims have an interpretation of the Quran that only allows music to be made with a human voice. Boston is one location where this is concentrated. This is a minority interpretation that would only affect 0.1% of our students based on my rough calculation.

I ask this question because even at 0.1% we should have already run into it.
